src/ray/gcs/redis_context.cc

```diff
@@ -171,6 +170,41 @@ RedisRequestContext::RedisRequestContext(instrumented_io_context &io_service,
   }
 }
 
+void RedisRequestContext::RedisResponseFn(struct redisAsyncContext *async_context,
+                                          void *raw_reply,
+                                          void *privdata) {
+  auto *request_cxt = (RedisRequestContext *)privdata;
```

Comment:
why do we use a c style case here, and a reinterpret_cast a line below? Maybe we can do it consistently

Reply:
Sure, I'll clean it up. This was just moved over verbatim from the pre-existing legacy code.